i18n/translation-en.json

```diff
@@ -167,6 +167,14 @@
       "precision": "Precision"
     }
   },
+  "state": {
```

Comment:
Can we have this object in `option` please? So it will look something like this:

```
  "option": {
    "annotationColor": {
      "StrokeColor": "Border color",
      "FillColor": "Fill color",
      "TextColor": "Text color"
    },
    "displayMode": {
      "layout": "Layout",
      "pageTransition": "Page transition"
    },
    "layout": {
      "cover": "Cover",
      "double": "Double",
      "single": "Single"
    },
    ....
   "state": {
    "accepted": "Accepted",
    "rejected": "Rejected",
    "completed": "Completed",
    "cancelled": "Cancelled",
    "set": "Set status:",
    "none": "None"
  }
}
```

Reply:
Resolved in dcb321c. Thank you!